src/ess/dream/data.py

```diff
@@ -278,5 +282,17 @@ def tof_lookup_table_high_flux() -> str:
 
```

Comment:
This docstring claims the old file was for 240 degrees already, but @celinedurniak mentioned on Slack it is 215. @nvaytet do you remember the history of this?

Reply:
I think I it's simply that the old docstring is wrong